### ISSUE TITLE
Porting Manager Teams Issue

### ISSUE THREAD

**Issue by ghost** [realworld666/SIncLib#6]
So, I've selected my 3 Update & Port teams (Day, Evening, Night), however whenever the port get put into my queue, it only chooses 1 out of the 3 teams to do the port instead of all three teams being assigned to the same port. I've highlighted via a red box of what i'm talking about. If you see on the right hand side of the screenshot, you can see only one team is assigned to the port, which means the port is only happening during those specific hours (day, evening, night), and not around the clock like i want it to. 

Is this the intended function?

![image](https://github.com/realworld666/SIncLib/assets/4575704/1f6c5b23-89f9-4488-bd8f-63b9df3350f2)

**Comment by realworld666** [comment]
Hi @BippyMiester 
Yes, this is the intended behaviour. To allow you to parallelize porting by having multiple teams.

**Comment by ghost** [comment]
Yes, but my teams are sorted by time of day. Most peoples teams are sorted by time of day. So, for only one team to port one thing, only 8 hours in a 24 hour day is kind of pointless. Its too slow. Can you add an option to add all teams to all the porting jobs? Like a simple checkbox that just adds all the teams?

**Comment by realworld666** [comment]
Hi @BippyMiester 
I've not forgotten about this. I've taken a break for a while and I'm starting to look at the mod again now. I'm currently in the middle of adding a manager for creating Addons so will circle back to this once that's done

**Comment by chadsimms** [comment]
> Hi [@BippyMiester](https://github.com/BippyMiester) I've not forgotten about this. I've taken a break for a while and I'm starting to look at the mod again now. I'm currently in the middle of adding a manager for creating Addons so will circle back to this once that's done

If you're still working on this - you can update the StartPortingJob() in PortingBehaviour.cs where you have:

> // add the job to the team
>  portingJob.AddDevTeam(team);

change it to:

> foreach (string teamName in PortingTeams)
> {
>     Team t = GameSettings.GetTeam(teamName);
>     if (t != null)
>         portingJob.AddDevTeam(t);
> }

**Comment by Jonesy-Gaming** [comment]
<img width="1614" height="892" alt="Image" src="https://github.com/user-attachments/assets/b7fcee0b-3f9b-4798-8b08-dcf38ae19adc" />
If you still working on this addon, then it could use some love to get it to work with current version of the game again😁

![Image](https://github.com/user-attachments/assets/baf6fb3e-ee36-47a8-880b-c189f68659b2)

**Comment by Jonesy-Gaming** [comment]
From patch notes 1.8.34:
Patched a security vulnerability in code mods (Note that mods using UnityEngine.PlayerPrefs will no longer load)

**Comment by realworld666** [comment]
This should be updated now
